### PR TITLE
Tag MsgPack.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [MsgPack.jl](https://github.com/kmsquire/MsgPack.jl)
Release: [v0.1.0](https://github.com/kmsquire/MsgPack.jl/releases/tag/v0.1.0)
Diff: [vs v0.0.5](https://github.com/kmsquire/MsgPack.jl/compare/1fbe1ed3a2f69763b787fede3526816efa9975fa...6b5f4ece0c7d31da4c762c136c89604707246cfd)
`requires` vs v0.0.5: 
```diff
--- 0.0.5/requires
+++ 0.1.0/requires
@@ -1,2 +1,2 @@
-julia 0.3
+julia 0.5
 Compat
```
cc: @kmsquire

### CODE REVIEW

File: MsgPack/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+Compat
```
Comment (tkelman) [inline MsgPack/versions/0.1.0/requires]: is this still needed? what minimum version implements the features you're using?

**Comment by kmsquire** [comment]
I've removed most of the need for Compat (still have some things recently introduced in 0.6), and updated on master.

I also enabled appveyor, and found and fixed some 32-bit bugs which showed up on the win32 build.  Will tag again later today.

**Comment by kmsquire** [comment]
Hmm... I had hoped attobot would open a new, corrected PR. Oh, well, I'll tag manually.

**Comment by simonbyrne** [comment]
See https://discourse.julialang.org/t/attobot-a-package-release-bot/494/11?u=simonbyrne

I can fix it by deleting the branch.

**Pull request closed without merging**